tests/Feature/Api/UserInfo/UserInfoApiTest.php

```diff
@@ -144,12 +141,25 @@ public function test_create_user_info_without_required_field()
             'twitter_url' => "https://twitter.com/12345"
         ];
 
-        $response = $this->json('PUT', "/api/v1/users/$user->id/info", $data);
+        $response = $this->json('POST', "/api/v1/users/$user->id/profile", $data);
+        $response->assertHeader('Content-Type', 'application/json');
         $response->assertStatus(400);
     }
 
     public function test_get_user_list_not_found()
     {
-        $this->json('GET', "/api/v1/user/9999/info")->assertStatus(404);
+        $this->json('GET', "/api/v1/user/9999/profile")->assertStatus(404);
```

Comment:
route is /api/v1/user/9999/profile, but method is test_get_user_list_not_found. user list? )